src/test/java/org/ambraproject/rhino/service/impl/SyndicationCrudServiceImplTest.java

```diff
@@ -0,0 +1,175 @@
+  }
+
+  @BeforeMethod
+  public void initMocks() throws IllegalAccessException, NoSuchFieldException {
+    mockSyndicationCrudService = applicationContext.getBean(SyndicationCrudService.class);
+    mockArticleCrudService = applicationContext.getBean(ArticleCrudService.class);
+    mockRuntimeConfiguration = applicationContext.getBean(RuntimeConfiguration.class);
+    mockJournalCrudService = applicationContext.getBean(JournalCrudService.class);
+    mockHibernateTemplate = applicationContext.getBean(HibernateTemplate.class);
+    mockMessageSender = applicationContext.getBean(MessageSender.class);
+  }
+
+  @Test
+  public void testGetSyndication() throws Exception {
+    final ArticleRevisionIdentifier revisionId = ArticleRevisionIdentifier.create("0", 1);
+    when(mockArticleCrudService.readRevision(revisionId)).thenReturn(createStubArticleRevision());
+    when(mockHibernateTemplate.execute(any())).thenReturn(createStubSyndication());
+
+    mockSyndicationCrudService.getSyndication(revisionId, "test");
+  }
+
+  @Test
+  public void testGetSyndications() throws Exception {
+    final ArticleRevisionIdentifier revisionId = ArticleRevisionIdentifier.create("0", 1);
+    when(mockArticleCrudService.readRevision(revisionId)).thenReturn(createStubArticleRevision());
+    when(mockHibernateTemplate.execute(any())).thenAnswer(new Returns(stubSyndications));
+
+    mockSyndicationCrudService.getSyndications(revisionId);
+  }
+
+  @Test
+  public void testUpdateSyndication() throws Exception {
+    final ArticleRevisionIdentifier revisionId = ArticleRevisionIdentifier.create("0", 1);
+    when(mockArticleCrudService.readRevision(revisionId)).thenReturn(createStubArticleRevision());
+    when(mockHibernateTemplate.execute(any())).thenReturn(createStubSyndication());
+
+    mockSyndicationCrudService.updateSyndication(revisionId, "test", "test", "test");
+
+    verify(mockHibernateTemplate).update(any(Syndication.class));
+  }
+
+  @Test
+  public void testCreateSyndication() throws Exception {
+    final ArticleRevisionIdentifier revisionId = ArticleRevisionIdentifier.create("0", 1);
+    when(mockArticleCrudService.readRevision(revisionId)).thenReturn(createStubArticleRevision());
+
+    mockSyndicationCrudService.createSyndication(revisionId, "test");
+
+    verify(mockHibernateTemplate).save(any(Syndication.class));
+  }
+
+  @Test
+  public void testReadSyndications() throws Exception {
+    when(mockRuntimeConfiguration.getQueueConfiguration()).thenReturn(stubConfig);
+    when(mockJournalCrudService.readJournal("test")).thenReturn(new Journal("test"));
+    when(mockHibernateTemplate.execute(any())).thenAnswer(new Returns(stubSyndications));
+
+    mockSyndicationCrudService.readSyndications("test", ImmutableList.of("test"));
+  }
+
+  @Test
+  @DirtiesContext
+  public void testSyndicateExistingSyndication() throws Exception {
+    final ArticleRevisionIdentifier revisionId = ArticleRevisionIdentifier.create("0", 1);
```

Comment:
Assigning `revisionId = ArticleRevisionIdentifier.create("0", 1)` could be done within the @BeforeMethod.

Reply:
Nice catch, thanks.